src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -243,7 +226,7 @@ public override int Run(AnalyzeOptions analyzeOptions)
             // Because of this, the return code bit for RuleNotApplicableToTarget is not
             // interesting (it will always be set).
 
-            return analyzeOptions.RichReturnCode
+            return analyzeOptions.RichReturnCode != null && analyzeOptions.RichReturnCode.Value
```

Comment:
>RichReturnCode [](http://example.com/codeflow?start=34&length=14)

Many SARIF SDK options are now nullable to allow them to be specified in a config file and then overridden on the command-line. This corner-case execution scenario will also require us to make a similar change to BinSkim. If we have time in 4.1 I'll make those updates in a follow-on PR. This PR is going in to enable the multithread analysis testing we need.